src/game_constants.h

```diff
@@ -88,6 +88,18 @@
 /** Maximum range at which ranged attacks can be executed */
 #define RANGE_HARD_CAP 60
 
+/** Player view distance FLOAT */
+#define PLAYER_VIEW_DISTANCE 60.0f
+
+/** Player view distance INTEGER */
+#define PLAYER_VIEW_DISTANCE_INT 60
+
+/** Kind of related... but not to view distance exactly? Changing this would change balance of things I think? */
+#define DISTANCE_FROM_CENTER 60
+
+/** Was used based off lighting but I'm not sure?*/
+#define LIGHT_SIXTY_F 60.0f
```

Comment:
This won't work right. You'd need to alter air transparency instead, otherwise you'll get a ton of side effects.